principles/work_in_the_open.md

```diff
@@ -0,0 +1,49 @@
+# Work in the open
+
+> *Our code is open as early as possible. We only go private if we really, really, **really** have to!*
+
+[Point 8](https://www.gov.uk/service-manual/service-standard/make-all-new-source-code-open) of the [Digital Service Standard](https://www.gov.uk/service-manual/service-standard) states
+
+> You must make all new source code open to meet point 8 of the Digital Service Standard.
+
+So you could develop your code in private, and release in time for a [Beta](https://www.gov.uk/service-manual/agile-delivery/how-the-beta-phase-works) assessment, which (other factors aside) would be sufficient to pass.
+
+However we believe there are more benefits to making code public from the beginning, rather than starting private and releasing later.
+
```

Comment:
Yes on the getting it updated - I need to follow that up now that its original author has left...

I think we should mention in this guide that an operational instruction exists - just something to signpost it for people rather than necessarily linking to it or anything like that.